test_vm/Cheffile

```diff
@@ -8,3 +8,4 @@ cookbook 'dbfit_test', :path => "vendor/cookbooks/dbfit_test"
 cookbook 'database', :git => "https://github.com/dbfit/database"
```

Comment:
Maybe a bit off-topic here, but some PRs have been recently merged into the upstream `database` git repo. Maybe it's already good enough and no need to customize the cookbook location.